src/manage.py

```diff
@@ -3,6 +3,10 @@
 import os
 import sys
 
+from dotenv import load_dotenv
+
+load_dotenv(override=True)
```

Comment:
@peymanslh, I think we should move these lines to the settings.py 
@hramezani what do you think? (In our company we put those lines to the settings.py file)